Directory.Build.props

```diff
@@ -3,7 +3,6 @@
     <AnalysisLevel>latest</AnalysisLevel>
     <AnalysisMode>AllEnabledByDefault</AnalysisMode>
     <Authors>Temporal</Authors>
-    <ContinuousIntegrationBuild Condition="'$(GITHUB_ACTIONS)' == 'true'">true</ContinuousIntegrationBuild>
```

Comment:
.NET 8 release (or MSBuild release) seems to now be properly setting deterministic source paths when this is set, which means it strips runtime paths which hurts us because we test with them.